featuretools/selection/selection.py

```diff
@@ -127,6 +132,19 @@ def remove_highly_correlated_features(feature_matrix, features=None, pct_corr_th
     fm_to_check = (feature_matrix[features_to_check]).select_dtypes(
```

Comment:
This is probably Amir's, but I didn't know about `select_dtypes()`, cool!

Reply:
@amirtis

Reply:
I think it was in featuretool's original code

Reply:
Oh, I haven't even noticed this is the featuretools fork <facepalm>